pom.xml

```diff
@@ -1204,6 +1204,12 @@
                 <artifactId>greenmail</artifactId>
                 <version>${greenmail.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.h2database</groupId>
+                <artifactId>h2</artifactId>
+                <version>${h2.db.version}</version>
```

Comment:
We have a h2database wso2 orbit bundle. Can't we use the same? That dependency is already there in the product-ei pom files.
            <dependency>
                <groupId>com.h2database.wso2</groupId>
                <artifactId>h2-database-engine</artifactId>
                <version>${wso2.h2.orbit.version}</version>
            </dependency>

Reply:
It cannot be used because the h2 version used in wos2 orbit is not supporting mv_store connection settings.